Prace_domowe/Praca_domowa2/Grupa4/Marcin Wilk/PD2.ipynb

```diff
@@ -0,0 +1,1563 @@
+{
```

Comment:
<p>A jakieś minusy TE?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>To nie jest główna wada TE. Zauważ, że jak grupujemy dane (jak Nike'i w jednym z pierwszych labów), to też tracimy dane, a chyba jest to dobre, prawda? (Unikamy overfittingu).</p><p> </p><p>A właśnie overfitting jest potencjalnym minusem TE, bo korzystamy z targetu</p>